ereolen_support/ereolen_support.helper.inc

```diff
@@ -192,6 +190,34 @@ function _ereolen_support_send_to_jira($form, &$form_state, $jiraUser) {
   return NULL;
 }
 
+function _ereolen_material_send_to_jira($form, &$form_state, $jiraUser) {
+  $data = [
+    'fields' => [
+      'project' => [
+        'id' => variable_get('jira_material_project_id'),
+      ],
+      'issuetype' => [
+        'id' => variable_get('jira_material_issue_type_id'),
+      ],
+      'summary' => preg_replace('/\s+/', ' ', check_plain(trim(substr($form_state['values']['ereolen_material_description'], 0, 30)))),
```

Comment:
Maybe you could use https://api.drupal.org/api/drupal/includes!unicode.inc/function/truncate_utf8/7.x here for the actual trimming, but the code here is also fine.